tests/client/test_auth.py

```diff
@@ -968,8 +967,7 @@ def test_build_metadata(
         revocation_options=RevocationOptions(enabled=True),
     )
 
-    assert metadata == snapshot(
-        OAuthMetadata(
+    assert metadata == OAuthMetadata(
```

Comment:
What is the error you were having?

Reply:
when I run `uv run pytest` I get the following error:
![image](https://github.com/user-attachments/assets/1bc1d7d4-cad1-490b-8f26-8ba57a4173a5)

As a fix I tried running:
`uv run pytest --inline-snapshot=fix tests/client/test_auth.py` but got the same error again, alternatively I tried using Is for parameterised variables, but that did not seem like fully correct either.